doc/content/sfr/subchannel/multiple_SCM_subassemblies/multi_assemblies.md

```diff
@@ -0,0 +1,119 @@
+                       computing_needs:Workstation
+                       transient:steady_state
+                       fiscal_year:2025
+                       sponsor:NEAMS
+                       institution:INL
+
+We include two example cases of coupling multiple subchannel models (`SCM`) with a 3D heat conduction model (`MOOSE`) of the wrapper and inter-wrapper. The wrapper is the hexagonal duct that contains the subchannel subassemblies and the inter-wrapper is the space in-between the ducts. A picture of the wrapper and inter-wrapper for the case of 19 subassemblies is shown in [inter-wrapper]
+
+!media subchannel/multiple_SCM_subassemblies/19inter.png
+    style=width:90%;margin-bottom:2%;margin:auto;
+    id=inter-wrapper
+    caption=Wrapper and inter-wrapper of 19 hexagonal subassemblies
+
+## Example Description
+
+One example models 7 subchannel subassemblies with a heat conduction model of the wrapper/inter-wrapper and the other one models 19 subchannel subassemblies with a heat conduction model of the wrapper/inter-wrapper. In both models SCM is run as a MultiApp and heat conduction as the main app. The wrapper material is steel and the inter-wrapper is liquid sodium. Both materials are treated as solids and only heat conduction is considered. In the 7 subassembly case  the central subassembly has twice the power of its neighbors. In the 19 subassembly case or assemblies have the same power.
+
+## Coupling scheme
+
+The coupling approach is that of domain decomposition. The main app calculates the temperature field in the wrapper and inter-wrapper regions. Based on the temperature field it calculates the linear heat-flux (W/m) on the inner surface of the wrapper (inner surface of the hexagonal ducts) and transfers that information to the equivalent duct model of SCM. The calculated linear heat-flux by the heat conduction model for the 7 assemblies case is shown in [heat-flux]
+
+!media subchannel/multiple_SCM_subassemblies/qprime.png
+    style=width:60%;margin-bottom:2%;margin:auto;
+    id=heat-flux
+    caption=Linear heat-flux on the inner ducts for the 7 assembly case. Heat conduction model.
+
+The linear heat-flux transfered to the duct model of SCM is shown in [heat-flux2].
+
+!media subchannel/multiple_SCM_subassemblies/qprime2.png
+    style=width:60%;margin-bottom:2%;margin:auto;
+    id=heat-flux2
+    caption=Linear heat-flux on the inner ducts for the 7 assembly case. SCM model.
+
+We have chosen to show two out of the seven SCM inner ducts. The center inner duct and one neighbor. In the 7 assembly case the central assembly has twice the power of its neighbors.
+Because of that heat flows from the high temperature assembly through the wrapper/inter-wrappe domain into the neighboring colder assemblies. Notice how the ranges of the values for heat-flux differ between the heat conduction model and the SCM models. This is because the cell size in SCM is larger in the lateral direction.
```

Comment:
```suggestion
For this reason, heat flows from the high temperature assembly through the wrapper/inter-wrapper domain into the neighboring colder assemblies. Notice how the ranges of the values for heat-flux differ between the heat conduction model and the SCM models. This is because the cell size in SCM is larger in the lateral direction.
```